### ISSUE TITLE
RFE: Accessablity Improvments for Dark Themes

### ISSUE THREAD

**Issue by StrikingVenom** [MegaMek/megamek#5892]
Currently, both MegaMek dark themes, _Flat Dark_ and _Flat Dracula_, use the same color set, with text color set as **#BBBBBB** while the background use **#3C3F41**, this results in a contrast ratio of **5.52:1**.  This contrast ratio is acceptable for many, even most, uses but fails the higher _**AAA**_ small text standard provided in the _**Web Content Accessibility Guidelines**_.  Moreover, the background on buttons and text boxes use the color **#4E5052**, and a contrast ratio of **4.21:1** which fails both the _**AA**_ and _**AAA**_ (WCAG) standard for small text, and the _**AAA**_ standard for large text. 
 
_**An incredibly straightforward first step for accessibility would be to change the text color to white.**_   Changing the text color from **#BBBBBB** to **#FFFFFF** would immediately bring the text-on-background contrast ratio from **5.52:1** to **10.6:1**, while the text-on-textbox contrast ratio would go from **4.21:1** to ** 8.09:1**.  Additonally, increasing font size, and optionally bolding the text, would also further improve accessibility.

While perhaps an unreasonable goal, enabling the end-user to easily create fully custom themes, with basic presets meeting different accessibility standards, would be the ideal end-goal.

The _**Web Accessibility Initiative**_ has a robust set of guidelines available online to enable better accessibility, and will be linked below, along with a free, web-based contrast checking tool. 

 

[_**Web Accessibility Initiative Quick Reference**_](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&currentsidebar=%23col_overview&showtechniques) 
 
[_**WebAIM Contrast Checker**_](https://webaim.org/resources/contrastchecker/)

**Comment by IllianiCBT** [comment]
Fully customizable themes is pretty much out of the realms of possibility as (insofar as I'm aware) we really don't have a GUI specialist on the team right now, so massive GUI overhauls (including themes) aren't really on the books.

However, what I would like to see is us bringing default font color into a MekHQ option, similar to what we did with Negative, Positive, and Warning text.
